css/styles.less

```diff
@@ -0,0 +1,830 @@
+@font-headings: 'Raleway';

```

Comment:
your styles are still a little off.
I added a bg image for you to see.
you can download the changes i made in
the safiyc-master branch i pushed on the adam repo.